programmers/난이도별/level01.자릿수_더하기/khw970421.js

```diff
@@ -0,0 +1,10 @@
+function solution(n)
+{
+    let a=[],sum=0;
+    a.push(String(n));  //숫자를 문자로 바꿔서 a배열에 넣는다.
+
+    a[0].split('').forEach(function(e){     //넣은것을 글자마다 분리하여 그때의 숫자를 더하여 계산
+        sum+=Number(e);
+    })
+    return sum;
```

Comment:
```suggestion
array method를 조금 더 적극적으로 활용해보세요

```suggestion
    return String(n).split('').map(Number).reduce((a, b) => a + b);
```
```

Reply:
forEach reduce map 와 같은 메서드 안에는 일반함수 대신 화살표 함수를 이용하는 것이 좋습니다.
this 문맥 때문에, 습관화 하는 것이 좋다고 합니다